src/keymap-manager.coffee

```diff
@@ -230,6 +230,10 @@ class KeymapManager
         console.warn("Encountered an invalid selector adding key bindings from '#{source}': '#{selector}'")
         return
 
+      unless typeof keyBindings is 'object'
+        console.warn("Encountered an invalid key binding when adding key bindings from '#{source}' '#{keyBindings}'")
+        return
```

Comment:
Could these be `continue` statement instead of return?  Although I know this already exist previously, just curious since this is returning, any keymaps that exist after the one that is invalid won't get loaded.

Also, curious if we should surface these warnings in the UI since currently, it's just in the console.